src/context-parser-handlebars.js

```diff
@@ -392,7 +392,7 @@ ContextParserHandlebars.prototype.analyzeAst = function(ast, contextParser, char
                 node.type === handlebarsUtils.RAW_EXPRESSION) {
 
                 // lookupStateForHandlebarsOpenBraceChar from current state before handle it
-                parser.setCurrentState(ContextParserHandlebars.lookupStateForHandlebarsOpenBraceChar[parser.state]);
+                parser.state = ContextParserHandlebars.lookupStateForHandlebarsOpenBraceChar[parser.state];
```

Comment:
i believe No. this particular line of code is meant to make the contextual analyzer work properly with a template as if the handlebars syntaxes are missing. you still remember what you wrote? :)